content/docs/command-reference/repro.md

```diff
@@ -51,7 +51,7 @@ There are a few ways to restrict what will be regenerated by this command: by
 specifying specific reproduction [`targets`](#options), or by using certain
 command [options](#options), such as `--single-item` or `--all-pipelines`.
 
-> Note that stages without dependencies nor outputs are considered [always
+> Note that stages without dependencies nor outputs are considered [always >
 > changed], so `dvc repro` always executes them.
```

Comment:
Idk why Prettier wants to do this. I'm going to ignore it and merge the original PR (which will close this one)...

Thoughts @julieg18 ? Just curious as it's unlikely we'll see this situation often.

Reply:
p.s. apparently we'll see this very often after #3287 was merged while ignoring this formatting.